cumulus/girderclient.py

```diff
@@ -117,18 +117,15 @@ def _upload_file(self, name, path, parent_id):
 
                 part = fp.read(chunk_size)
 
-                m = MultipartEncoder(
-                    fields=[('uploadId',  upload_id),
-                            ('offset', str(uploaded)),
-                            ('chunk', (name, part, 'application/octet-stream'))]
+                params['uploadId'] = upload_id
+                params['offset'] = uploaded
 
-                )
 
                 headers = self._headers.copy()
                 headers['Content-Type'] = m.content_type
```

Comment:
headers['Content-Type'] = m.content_type 

is wrong. You deleted m for the MultipartEncoder. It should be

headers['Content-Type'] = 'application/octet-stream'

Reply:
This should fix the failing test.